### PR TITLE
smoother progress bar

### PR DESCRIPTION
**Pull request by valpinkman**

### CODE REVIEW

File: src/components/base/Button/index.js [hunk 1]
```diff
@@ -221,7 +221,7 @@ class Button extends PureComponent<
     const onClickHandler = e => {
       if (onClick) {
         if (event) {
-          track(event, eventProperties)
+          // track(event, eventProperties)
```
Comment (gre) [inline src/components/base/Button/index.js]: why the change?

Comment (valpinkman) [reply src/components/base/Button/index.js]: oh sorry, we needed to comment that for storybook to work, I will fix this
